src/js/components/RoomBox.js

```diff
@@ -47,4 +47,4 @@ var RoomBox = React.createClass({
     }
 });
 
```

Comment:
Sorry about the whitespaces deletion, my editor (Atom) has its way. You should know though, that leaving such whitespaces is considered a bad practice by most.